test/integration/test_shrink.py

```diff
@@ -179,4 +205,23 @@ def test_shrink_with_extras(self):
         self.assertTrue(settings[self.target]['settings']['index']['routing']['allocation'][allocation_type][key] == value)
         self.assertTrue(settings[self.idx]['settings']['index']['routing']['allocation']['require']['_name'] == '')
         self.assertEqual(settings[self.target]['settings']['index']['codec'], 'best_compression')
-        self.assertTrue(self.client.indices.exists_alias(index=self.target, name='my_alias'))
+        self.assertTrue(self.client.indices.exists_alias(index=self.target, name='my_alias')
```

Comment:
You're missing the trailing `)` here, and that's why the test is failing.  It was there before, so you probably removed it by accident.

Reply:
Oh, sorry. I might delete when fixing some conflicts in merge. I will fix it.